src/app/view/ChessBoardView.ts

```diff
@@ -1,21 +1,42 @@
-//Nie przywiazujcie sie do tego kodu. Mozecie sprobowac wszystko pozmieniac / zrobic inaczej.
+export class Square {
+  constructor(private position_x: number, private position_y: number, private color: SquareColor) {}
 
-export type Square = {
-  id: string;
-};
+  setHtmlElement(): HTMLElement {
+    const squareElement = document.createElement('div');
+    squareElement.id = this.mapSquarePosition();
+    squareElement.classList.add('square');
+    squareElement.classList.add(`square--${this.color}`);
+    return squareElement;
+  }
 
-export enum Side {
-  WHITE = 'WHITE',
-  BLACK = 'BLACK',
+  mapSquarePosition(): string {
+    const columns = 'abcdefgh'.split('').reverse();
```

Comment:
jak w logice mamy drukowany literami nazwy kolumn to może tutaj też lepiej tak nazwać? Ale to pierdoła :-P